LICENSE

```diff
@@ -1,3 +1,40 @@
+   =========================================================================
```

Comment:
These may be unnecessary (the git commit message is clear about origin). However, if we want to retain them, this should go into a NOTICES file (there is one in sawtooth-core for example).